docs/graphback-datasync.md

```diff
@@ -0,0 +1,230 @@
+---
+id: graphback-datasync
+title: Graphback Data Synchronization package
+sidebar_label: Setting up Data Synchronization using Graphback
+---
+
+The `@graphback/datasync` package consisting of the Data Synchronization Schema plugin and compatible data sources, provides out of the box Data Synchronization strategies for GraphQL clients with offline functionality e.g. [Offix](https://offix.dev). Currently this plugin **only** supports MongoDB data sources, with support for other kinds of data sources coming in a future release.
+
+## Motivation
+
+The raison d'être of the `@graphback/datasync` package is to provide additional out of the box functionality to offline-first GraphQL clients, mainly targeting [Offix](https://offix.dev). To that end, it has two main goals to fulfill:
+
+1. Provide Delta Queries: Delta queries can be used by GraphQL clients to refresh changes in data between periods of connectivity outages. This is functionally complete for MongoDB with support for other data sources and strategies under development.
+2. Provide Server-side Conflict Resolution capabilities: For mutations that are applied offline, this provides the server with the ability to resolve conflicts between server and client side data. This functionality is yet to be implemented.
+
+## Installation
+
+The Graphback Data Synchronization package provides schema plugins as well as data sources for all supported data synchronization patterns.
+
+```bash
+npm install @graphback/datasync
+```
+
+## Usage
+
+There are two steps to implementing a data synchronization strategy:
+
+- Add metadata to schema
+- Use the relevant plugin and data source
+
+Currently the supported strategies are:
+
+- Soft deletes with delta queries for MongoDB
+
+## Soft Deletes with delta queries
+
+1. ### Sprinkle some metadata in your schema
+
+Add the `versioned` and `delta` markers to your model(s) in your GraphQL SDL:
```

Comment:
Let's use `annotations` instead of `markers` in documentation.
/cc @craicoverflow @wtrocki